src/screens/SendConfirmScreen.js

```diff
@@ -59,21 +58,35 @@ class SendConfirmScreen extends React.Component {
     this.amountAndToken = `${hathorLib.helpers.prettyValue(this.amount)} ${this.token.symbol}`;
   }
 
+  /**
+   * In case we can prepare the data, open send tx feedback modal (while sending the tx)
+   * Otherwise, show error
+   *
+   * @param {String} pinCode User PIN
+   */
   executeSend = (pinCode) => {
-    // show loading modal
-    this.setState({
-      modal:
-        // eslint-disable-next-line react/jsx-indent
-        <FeedbackModal
-          icon={<Spinner />}
-          text={t`Your transfer is being processed`}
-        />,
-    });
-    this.props.sendTx(this.amount, this.address, this.token, pinCode).then(
-      this.onSuccess, this.onError
-    );
+    const ret = this.props.sendTx(this.amount, this.address, this.token, pinCode);
+    if (ret.success) {
+      // show loading modal
+      this.setState({
+        modal:
+          // eslint-disable-next-line react/jsx-indent
+          <SendTransactionFeedbackModal
+            text={t`Your transfer is being processed`}
```

Comment:
I think we should say: "Your transaction is being resolved...".
We may also add something like: "This is an anti-spam measure to prevent abuse."

What do you think?

Reply:
I think the app is for more common use and some technical information is not always good. Besides that, I think adding the sentence could become too much text. I could change 'processed' to 'resolved' but I still don't know which one is the best.

Reply:
Agree with @pedroferreira1. I actually think we could change to "Your transaction is being processed", but I also don't mind the way it is.

Reply:
I agree with you, but I'd like to differentiate a "mining slowness" from a "propagation slowness". How can we do it without complicating the UX?

Maybe we can just say "Preparing your transaction..." or "Mining your transaction..." or "Resolving your transaction", and add a link for further information.

Reply:
Where would we add this link for further information? I think we don't have where to put it. RIght now we are already adding a text "Propagating transaction to the network", when it moves from mining to propagating, so it already differentiate both action, however the propagation is so fast that is not even noticed.

Reply:
Ok. I agree that we have a small box, and we still don't have any explanations in our website. I think we can create another issue to do it later and move on.

Reply:
Done